src/provider/provider_os_memory.c

```diff
@@ -83,28 +86,64 @@ static void os_store_last_native_error(int32_t native_error, int errno_value) {
 
 static umf_result_t nodemask_to_hwloc_nodeset(const unsigned *nodelist,
```

Comment:
this function should be called 'nodelist_to_hwloc_nodeset' I guess

Reply:
done?